tests/by-util/test_mv.rs

```diff
@@ -716,6 +718,168 @@ fn test_mv_update_option() {
     assert!(!at.file_exists(file_b));
 }
 
+#[test]
+fn test_mv_arg_update_none() {
+    let (at, mut ucmd) = at_and_ucmd!();
+
+    let file1 = "test_mv_arg_update_none_file1";
+    let file2 = "test_mv_arg_update_none_file2";
+
+    at.touch(file1);
+    at.touch(file2);
+
+    let file1_content = "file1 content\n";
+    let file2_content = "file2 content\n";
+
+    at.append(file1, file1_content);
+    at.append(file2, file2_content);
```

Comment:
You can simplify the tests a bit by using 
```rust
at.write(file1, file1_content);
```
instead of

```rust
at.touch(file1);
at.append(file1, file1_content);
```